examples/c/mass_spring_qpdunes.c

```diff
@@ -145,8 +146,10 @@ int main() {
 
     printf("\ninf norm res: %e, %e, %e, %e\n", res[0], res[1], res[2], res[3]);
 
-    printf("\nSolution time for %d dual Newton iterations, averaged over %d runs: %5.2e seconds\n\n\n",
-        -1, NREP, time);
+    ocp_qp_info *info = (ocp_qp_info *)qp_out->misc;
+
+    print_ocp_qp_info(info);
```

Comment:
does this function also print the number of iterations now? Would be nice

Reply:
it indeed prints out the number of iterations